### ISSUE TITLE
navigator 'busy' on quit menu selection but no activity visible

### ISSUE THREAD

**Issue by qt3m45su0najc7** [conda/conda#8288]
## Current Behavior
After updating environment with latest Anaconda in Anaconda navigator, updating seems to complete and no further activity seems to be occurring.  When I  menu>quit, I get a dialog messge saying navigator is still busy.  I don't know if it is still busy or what it is busy doing.  At the end, I don't know if Anaconda is updated or not.

### Steps to Reproduce
1. Open Navigator
1. Select default environment
1. Click 'anaconda' for updating
1. Click 'apply'. Dialog displays showing packages which will be updated.
1. Click 'apply' to accept.  Progress bar displays in navigator window then clears after activity (presumably) ceases.
1. On navigator menu, click 'file>quit'. Warning dialog appears showing Navigator is still busy.  No other indications that it is busy or what it is doing.

## Expected Behavior
I think I should see what activity is incomplete and the progress bar should not be cleared if previous actions have not been completed.

## Environment Information
<summary><code>

     active environment : base
    active env location : C:\Program Files (x86)\Microsoft Visual Studio\Shared\Anaconda3_64
            shell level : 1
       user config file : C:\Users\albert\.condarc
 populated config files : C:\Users\albert\.condarc
          conda version : 4.5.12
    conda-build version : 3.10.5
         python version : 3.6.5.final.0
       base environment : C:\Program Files (x86)\Microsoft Visual Studio\Shared\Anaconda3_64  (read only)
           channel URLs : https://repo.anaconda.com/pkgs/main/win-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/win-64
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/win-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/pro/win-64
                          https://repo.anaconda.com/pkgs/pro/noarch
                          https://repo.anaconda.com/pkgs/msys2/win-64
                          https://repo.anaconda.com/pkgs/msys2/noarch
          package cache : C:\Program Files (x86)\Microsoft Visual Studio\Shared\Anaconda3_64\pkgs
                          C:\Users\albert\AppData\Local\conda\conda\pkgs
       envs directories : C:\Users\albert\AppData\Local\conda\conda\envs
                          C:\Program Files (x86)\Microsoft Visual Studio\Shared\Anaconda3_64\envs
                          C:\Users\albert\.conda\envs
               platform : win-64
             user-agent : conda/4.5.12 requests/2.18.4 CPython/3.6.5 Windows/10 Windows/10.0.17134
          administrator : False
             netrc file : None
           offline mode : False
</code>
</summary>

## Selecting anaconda package
![anacondanavigatorbug 1a](https://user-images.githubusercontent.com/465722/52813448-69689a00-305f-11e9-9e26-e30e5f86794d.png)

## Confirmation of updates which will be applied

![anacondanavigatorbug 1b](https://user-images.githubusercontent.com/465722/52813445-68d00380-305f-11e9-9e2b-12a2431ac317.png)

## Dialog which indicates actions are still pending

Note in the status info bar at the bottom there is no progress bar.

![anacondanavigatorbug 2a](https://user-images.githubusercontent.com/465722/52813446-68d00380-305f-11e9-8c97-d6664ec18e03.png)

**Comment by PeteDiMarco** [comment]
I am seeing the same problem.  I just installed Anaconda Navigator 1.9.7 on Kubuntu Linux 18.04.2 LTS.  When I clicked on _base (root) -> Open Terminal_, nothing happened.  After several retries I tried to quit the application, but I got the message "Anaconda Navigator is still busy."
The anaconda-navigator process is not consuming a lot of CPU time.  Here are the error messages I'm seeing (I started it from the command line):
WARNING: The conda.compat module is deprecated and will be removed in a future release.
/home/bob/anaconda3/lib/python3.7/site-packages/anaconda_navigator/api/conda_api.py:1364: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f)
2019-07-24 10:08:39,049 - ERROR download_api._download:234
Invalid url https://www.anaconda.com/wp-content/uploads/2017/05/Webinar20-20Three20Ways20to20Move20your20Data20Science20Projects20to20Production.png

QXcbConnection: XCB error: 3 (BadWindow), sequence: 961, resource id: 39846146, major code: 40 (TranslateCoords), minor code: 0

QXcbConnection: XCB error: 3 (BadWindow), sequence: 11530, resource id: 39846210, major code: 40 (TranslateCoords), minor code: 0
QXcbConnection: XCB error: 3 (BadWindow), sequence: 12662, resource id: 39846213, major code: 40 (TranslateCoords), minor code: 0
QXcbConnection: XCB error: 3 (BadWindow), sequence: 13052, resource id: 39846216, major code: 40 (TranslateCoords), minor code: 0
QXcbConnection: XCB error: 3 (BadWindow), sequence: 13567, resource id: 39846219, major code: 40 (TranslateCoords), minor code: 0
QXcbConnection: XCB error: 3 (BadWindow), sequence: 15165, resource id: 39846225, major code: 40 (TranslateCoords), minor code: 0
QXcbConnection: XCB error: 3 (BadWindow), sequence: 16146, resource id: 39846253, major code: 40 (TranslateCoords), minor code: 0
QXcbConnection: XCB error: 3 (BadWindow), sequence: 16547, resource id: 39846256, major code: 40 (TranslateCoords), minor code: 0

**Comment by felipeqq2** [comment]
Same problem here, I'm afraid of shutting down the navigator and interrupt some important process.

**Comment by tjquinn1** [comment]
has anyone made progress with this? I now get it with pretty much anything I do. I tried creating a new env and it happens. I also tried updating the navigator itself and it also happens.

**Comment by Zunaira-Shafaq** [comment]
I had the same issue here. I was afraid to quit the anaconda navigator because it showed the message that Anaconda is still busy. This happened after I click on the "Yes" button to the update application dialogue box "Do you wish to update to Anaconda Navigator 2.0.1? but no activity visible.
So, I tried to manually update the application, which successfully gets me rid of this problem.

**First Step** 
From this [link](https://docs.anaconda.com/anaconda/navigator/update-navigator/), I followed the alternate method as:
----------------------------------------------------------------------
Open the Anaconda prompt/Terminal:
Run this command to deactivate conda:
`conda deactivate`
Then run this command to update Navigator:
`conda update anaconda-navigator`
----------------------------------------------------------------------
This will hopefully update your anaconda navigator. 

**Second Step** 
Then you have to start the anaconda application manually from the terminal. Help from this [link](https://docs.anaconda.com/anaconda/install/verify-install/).
----------------------------------------------------------------------
Open terminal and run this command:
`anaconda-navigator`
----------------------------------------------------------------------
This will start your updated anaconda application.

You just need to do this one time. After this, you can start your application by using desktop shortcut. Hopefully this method will resolve your issue.

**Comment by qt3m45su0najc7** [comment]
Thank you Zunaira.  Your explanation and especially the link to anaconda docs resolves the issue.

Initially I used **windows terminal** to run the update, but consistently got an error downloading the json file.  I was able to follow the referenced json file link in the browser without issue.  I switched to **command prompt**, and the download and update proceeded without fail.  I did not expect a difference in behavior.
